pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -431,6 +430,34 @@ func (r *ReconcileInfinispan) Reconcile(request reconcile.Request) (reconcile.Re
 		default:
 			reqLogger.Info("Cluster NOT exposed. Unsupported type?", "ExposeType", infinispan.Spec.Expose.Type)
 		}
+	} else if err != nil {
```

Comment:
please check this snippet, seems to me a dup of lines 287+

Reply:
Yes, thanks. This was regression from complex merge.